deployments/workloads.go

```diff
@@ -121,6 +124,13 @@ func (k Workloads) Deploy(c *kubernetes.Cluster, ui *ui.UI, options kubernetes.I
 		return err
 	}
 
+	s := ui.Progress("Warming up cluster with builder image")
```

Comment:
@andreas-kupries this message doesn't print :confused: any idea?

Reply:
Crank the verbosity level.

Reply:
Spinners show only the dots at the base level, and not their message.

Reply:
For things where I always want a line I use
```
		c.ui.Note().KeeplineUnder(1).Msg("Provisioning...")
		s := c.ui.Progressf("Provisioning")
		defer s.Stop()
```
(CreateService in the wait for provision branch)

Reply:
Done with #05053b1